### PR TITLE
Fix abuse triggering

### PR DESCRIPTION
**Pull request by yontalcar**

### CODE REVIEW

File: test/run-tests [hunk 1]
```diff
@@ -106,9 +106,20 @@ def handle_transient_httperrors(error):
...
+
+        if "abuse detection" in github_msg:
+            timeout = transient_error_wait_time.next()
+            print(
+                ">>> We have triggered abuse detection, waiting {} s...".format(timeout)
+            )
+            time.sleep(timeout)
+            return True
+
+        if github_msg:
```
Comment (tyll) [inline test/run-tests]: How about always printing the error message, i.e. moving this code before the abuse detection handling? Having the raw error message might be useful for debugging.

Comment (yontalcar) [reply test/run-tests]: Done

**Comment by yontalcar** [comment]
@tyll: re required checks: I guess "continuous-integration/travis-ci" was superseded by "Travis CI - Pull Request", wasn't it?

**Comment by tyll** [comment]
> @tyll: re required checks: I guess "continuous-integration/travis-ci" was superseded by "Travis CI - Pull Request", wasn't it?

yes, I fixed it.

**Pull request merged**